megamek/data/images/hexes/saxarba/Includes/Pavement.tileinc

```diff
@@ -0,0 +1,78 @@
+# Pavement
+
+# Unfortunately super pavements will interfere too heavily with terrain shadows
+# until a better system is found, this is reverted to the old pavement as base
+## INACTIVE: Using super for pavement instead of base because of the ground_fluff use
+## super terrains will only find exact matches while base terrain
+## will find best matches; super avoids having unwanted terrain images
+## Pavement should be included very early as it should be underneath everything (draw first)
+
+# pavement:1 is saxarba's pavement
+## pavement:1;ground_fluff:1 is SirMegaV's pavement
+## pavement:1;ground_fluff:2 is saxarba's pavement urban garden variation
+# pavement:2 is saxarba's pavement urban garden variation (should not use terrain level)

```

Comment:
I have commented the two lines with ## further with "(not implemented currently)". The other two (thus: all pavement) are actually unchanged over previous versions of saxarba because of the shadows problem detailed above those lines. I have various solutions in mind, maybe I'll get around to fixing this...